src/components/UpdatePollTitle/updatePollTitle.js

```diff
@@ -0,0 +1,64 @@
+import { useState, useEffect } from "react";
+import { InputGroup, FormControl, Button, Modal } from "react-bootstrap";
+import { useDispatch, useSelector } from "react-redux";
+import { useNavigate, useParams } from "react-router-dom";
+import updatePollTitle, { emptyUpdatePollTitleDetailsSuccessStatus } from "../../redux/updatePollTitle/actions/updatePollTitle"
+import "./updatePollTitle.css"
+
+const UpdatePollTitle = () => {
+  const { pollTitle, pollId } = useParams();
+  const updatPollTitleDetails = useSelector(state => state.updatePollTitleDetails)
+  const userLoginDetails = useSelector((state) => state.login.userLogin);
+  const dispatch = useDispatch()
+  const navigate = useNavigate()
+  const [title, setTitle] = useState(pollTitle)
+  const [error, setError] = useState("")
+
+  const handleTitleChange = (event) => {
+    setTitle(event.target.value);
+    setError("");
+  };
+
+  const handleUpdate = () => {
+    const regex = /^.{8,}$/;
+    if (!regex.test(title)) {
```

Comment:
make a helper function for this type of validation and use from there @PuniaMohit